test/tools/test_checkimports.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+import sys
+from contextlib import contextmanager
+from os import path
+sys.path.append(path.abspath(path.join(path.dirname(__file__),
+                                       '..', '..', 'tools')))
+try:
+  from StringIO import StringIO
+except ImportError:
+  from io import StringIO
+
+from checkimports import is_valid
+
+@contextmanager
+def captured_output():
+    tmp_out, tmp_err = StringIO(), StringIO()
+    old_out, old_err = sys.stdout, sys.stderr
+    try:
+        sys.stdout, sys.stderr = tmp_out, tmp_err
+        yield sys.stdout, sys.stderr
+    finally:
+        sys.stdout, sys.stderr = old_out, old_err
```

Comment:
```suggestion
  tmp_out, tmp_err = StringIO(), StringIO()
  old_out, old_err = sys.stdout, sys.stderr
  try:
    sys.stdout, sys.stderr = tmp_out, tmp_err
    yield sys.stdout, sys.stderr
  finally:
    sys.stdout, sys.stderr = old_out, old_err
```

Reply:
Perhaps `tmp_out.close()` and `tmp_err.close()` so we don't leak them?

Or indent this block of code under: `with StringIO() as tmp_out, StringIO() as tmp_err:`

Reply:
Added `tmp_out.close()` and `tmp_err.close()`.